src/styles/main.css

```diff
@@ -6,7 +6,7 @@
 }
 
 @font-face {
-  font-family: 'Gotham Rounded';
+  font-family: "Gotham Rounded";
```

Comment:
There are still double quotes in some places. Please, roll them back to single quotes.